charts/topolvm/README.md

```diff
@@ -85,6 +85,7 @@ See [Getting Started](https://github.com/topolvm/topolvm/blob/topolvm-chart-v15.
 | lvmd.prometheus.podMonitor.namespace | string | `""` | Optional namespace in which to create PodMonitor. |
 | lvmd.prometheus.podMonitor.relabelings | list | `[]` | RelabelConfigs to apply to samples before scraping. |
 | lvmd.prometheus.podMonitor.scrapeTimeout | string | `""` | Scrape timeout. If not set, the Prometheus default scrape timeout is used. |
+| lvmd.securityContext.privileged | bool | `true` |  |
```

Comment:
You shouldn't edit this file manually. Please run helm-docs instead. See also [this document](https://github.com/topolvm/topolvm/tree/main/charts/topolvm#update-readme).